src/dokan/options.cc

```diff
@@ -115,6 +117,23 @@ int parse_args(
     } else if (ceph_argparse_witharg(args, i, &win_vol_serial,
                                      "--win-vol-serial", (char *)NULL)) {
       cfg->win_vol_serial = std::stoul(win_vol_serial);
+    } else if (ceph_argparse_witharg(args, i, &max_path_len,
+                                     "--max-path-len", (char*)NULL)) {
+      unsigned long max_path_length = std::stoul(max_path_len);
+
+      if (max_path_length > 32767) {
```

Comment:
just a minor nit: the commit message says that the max value is 4096, however the actual limit is 32767.